packages/@aws-cdk/aws-eks/README.md

```diff
@@ -885,13 +920,16 @@ cluster.addManifest('hello-kub', service, deployment);
 
 The following example will deploy the resource manifest hosting on remote server:
 
-```ts
+```text
+// This example is only available in TypeScript
+
 import * as yaml from 'js-yaml';
 import * as request from 'sync-request';
 
+declare const cluster: eks.Cluster;
 const manifestUrl = 'https://url/of/manifest.yaml';
 const manifest = yaml.safeLoadAll(request('GET', manifestUrl).getBody());
-cluster.addManifest('my-resource', ...manifest);
+cluster.addManifest('my-resource', manifest);
 ```
```

Comment:
This example doesn't work because of the imports to `js-yaml` and `sync-request` which seem important for this particular example. How do you feel about doing it this way (I stole the idea from ec2).

Reply:
That's fine :)